esp/bindings/http/platform/httptransport.cpp

```diff
@@ -1915,11 +1915,16 @@ int CHttpRequest::receive(IMultiException *me)
     return 0;
 }
 
-ISpan * CHttpRequest::createServerSpan()
+ISpan * CHttpRequest::createServerSpan(const char * serviceName, const char * methodName)
 {
     //MORE: The previous code would be better off querying httpHeaders...
+    StringBuffer spanName;
+    spanName.append(serviceName);
+    if (!isEmptyString(methodName))
+        spanName.append("/").append(methodName);
```

Comment:
It's prob fine, but I wonder if the / delimiter could cause problems downstream. (prometheous, grafana, elastic will likely use this value in UI widgets). I didn't find specific rules on span nomenclature or even attribute naming rules which excludes the slash, but I'd consider another delimiter

Reply:
One of the examples on the open telemetry recommendations for span name conventions (https://github.com/open-telemetry/opentelemetry-specification/blob/v1.26.0/specification/trace/api.md#span) uses a / : "get_account/{accountId}" - so I think it can be considered safe.